Ws-Peroth/20211007/TileMapGenerator.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class TileMapGenerator : MapGeneratorBSP
+{
+    [SerializeField]
+    protected List<Sprite> tileSprites = new List<Sprite>(); // Tilemap Sprites
+
+    [SerializeField]
+    private GameObject _tile;   // Dummy Tile Object
```

Comment:
```suggestion
    private GameObject tile;   // Dummy Tile Object
```

`MapInitializer` 클래스에서는 `isDebugMode` 라는 필드가 protected/SirializeField인데 언더바가 안붙어있고 해당 필드는 언더바가 붙어있습니다. 코드 컨벤션을 통일해주세요.

Reply:
C# 코딩 표준 관련하여 알아보다 본 것인데, 제 스크립트에 있는 `protected` 멤버들을 `private`으로 선언한 후에 `protected` 프로퍼티로 공개하는게 좋을까요?
```cs
// ex
    [SerializeField]
    private bool _isDebugMode;

    protected bool IsDebugMode
    {
        get => _isDebugMode;
        set => _isDebugMode = value;
    }
```
@sonohoshi